### PR TITLE
Fix filters2

### PR DESCRIPTION
**Pull request by muselesscreator**
**TL;DR -**
Filters were not updating query params

JIRA: [JIRA-5696](https://openedx.atlassian.net/browse/JIRA-5696)

**What changed?**
Filters all update query params

**Developer Checklist**
- [x] Test suites passing
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**
update filters, verify they update the URI.
refresh and make sure they all stick.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: src/data/actions/filters.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const initializeFilters = ({
     assignmentGradeMax,
     courseGradeMin,
     courseGradeMax,
-    includeCourseRoleMembers,
```
Comment (jansenk) [inline src/data/actions/filters.js:34]: What does this do?

**Review by nsprenkle — APPROVED** [review]
Beautiful. Have verified locally :shipit: 

_Note:_ Right now the search box doesn't save to the query params (and maybe it shouldn't anyways), but I wrote a separate ticket this morning to note that.

**Review by jansenk — COMMENTED** [review]
Tested locally, looks good but I still am seeing the bug WRT/ cohorts

File: src/data/actions/filters.js (hunk 1 above)
Comment (muselesscreator) [reply src/data/actions/filters.js:34]: clears a complaint that the value is being passed from urlQuery as string rather than boolean.
